coverage_badge/__main__.py

```diff
@@ -49,23 +49,26 @@ def get_total():
     cov.load()
     total = cov.report(file=Devnull())
 
-    class Precision(coverage.results.Numbers):
-        """
-        A class for using the percentage rounding of the main coverage package,
-        with any percentage.
+    if hasattr(coverage.results.Numbers, 'set_precision'):
```

Comment:
Maybe add a comment?

```suggestion
    if hasattr(coverage.results.Numbers, 'set_precision'):  # Coverage <=5
```